### ISSUE TITLE
Gallery snippet does not pass correct value for Galleriffic option: enableKeyboardNavigation=`0`

### ISSUE THREAD

**Issue by nicboyde** [splittingred/Gallery#275]
It instead passes "0", so the condition can never be set to false and keyboard navigation cannot be turned off - rendering forms on the same screen useless. Spaces are intercepted  by the javascript.

Commenting-out lines 938-941 of /assets/components/gallery/packages/galleriffic20/js/jquery.galleriffic.js (which monitor the position of the space bar) is a fix, but it should be possible to just turn it off. The galleriffic documentation doesn't mention this optional setting, but it is included.

**Comment by jpdevries** [comment]
@nicboyde thanks for reporting this. Gallery development has been moved to https://github.com/modxcms/Gallery/
